packages/ocvImpact/R/run_country_scenario.R

```diff
@@ -59,7 +59,12 @@ run_country_scenario <- function(
   montagu_cache[["country_agePop"]] <- import_country_agePop(modelpath, country, montagu_cache, redownload = FALSE)
   montagu_cache[["country_lifeExpectancy"]] <- import_country_lifeExpectancy(modelpath, country, montagu_cache, redownload = FALSE)
 
-  vacc_alloc <- allocate_vaccine(datapath, modelpath, country, scenario, montagu_cache, ...) #the changes start from here
+  if (config$vacc$targeting == "custom"){
+    ## use the custom-made targeting table for the 'custom' targeting strategy
+    vacc_alloc <- readr::read_csv(paste0(datapath,"/custom_targeting.csv")) ##if the custom targeting table is inside /input_data
```

Comment:
We should make the filename a config option, as we would expect the filenames to differ with each custom run. We could also then remove the datapath specification here, as it would be specified explicitly from the config, relative to the base repository path